system/core/dynamic/BeanPopulator.cfc

```diff
@@ -442,4 +469,22 @@ Description :
 		<cfreturn createObject("component","coldbox.system.core.util.Util")/>
 	</cffunction>
 
+	<!--- Get the Target's properties as a struct with a key and value that are the same for easy access and setting of the key's casing --->
+	<cffunction name="getTargetPropertiesAsStruct" access="private" output="false" returntype="struct" hint="Return the property names for the target object as a struct with the key the same as the value">
+		<cfargument name="target" required="true" type="any" />
+
+		<cfscript>
+			// get array of properties
+			var stopRecursions= [ "lucee.Component", "railo.Component", "WEB-INF.cftags.component" ];
```

Comment:
Likely. Just wasn't sure as I saw this line in use in the other instance where the `getInheritedMetaData()` method was used so I thought it might be needed. 

Just haven't had the time to work on some tests yet.